python/tk_multi_snapshot/snapshot.py

```diff
@@ -430,10 +430,8 @@ def show_snapshot_dlg(self):
             msg = "Unable to snapshot!\n\nPlease save the scene as a valid work file before continuing"
             QtGui.QMessageBox.information(None, "Unable To Snapshot!", msg)
 
-            # try to launch "Shotgun Save As" command if we have it:
-            save_as_cmd = tank.platform.current_engine().commands.get(
-                "Shotgun Save As..."
-            )
+            # try to launch "SG Save As" command if we have it:
+            save_as_cmd = tank.platform.current_engine().commands.get("SG Save As...")
```

Comment:
Hey @zamu5 -- here we're going to need to make it backwards compatible. Have a look at the lines immediately below this and you'll see that the command name was changed once in the past, and we fall back to that if we don't find "Shotgun Save As...". Could you do the same thing here and look for "SG Save As..." first, "Shotgun Save As..." second, and "Tank Save As..." last?